src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -394,6 +399,16 @@ class ConsignmentRouteSpec extends AnyFlatSpec with Matchers with TestRequest wi
     response.errors.head.extensions.get.code should equal("NOT_AUTHORISED")
   }
 
+  "startUpload" should "add the upload status and update the parent folder" in {
```

Comment:
Should have noticed this earlier, sorry, but what about the case where `startUpload` is called twice? The second one should be rejected, like a second `createFiles` is now.

Reply:
Good point, I've set it to reject if the consignment status for "Upload" is "InProgress" or "Complete". I've added a couple of tests to check for it.